pkg/forth/forth_test.go

```diff
@@ -43,6 +43,8 @@ var forthTests = []forthTest{
 	{val: "5 5 + hostbase", res: "", err: strconv.ErrSyntax, empty: true},
 	{val: "1 1 '+ newword 1 1 '+ newword", res: "", err: ErrWordExist, empty: true},
 	{val: "1 4 bad newword", res: "", err: ErrNotEnoughElements, empty: false},
+	{val: "1 %d printf", res: "1\n", empty: true},
```

Comment:
looks like we have a bug in tests, for example if you change 
`{val: "1 %d printf", res: "3\n", empty: true},`

it will pass because we ignore res here:
`if res == tt.res || errors.Is(err, tt.err) {`